api4/group_local.go

```diff
@@ -3,7 +3,113 @@
 
 package api4
 
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
 func (api *API) InitGroupLocal() {
-	api.BaseRoutes.Channels.Handle("/{channel_id:[A-Za-z0-9]+}/groups", api.APILocal(getGroupsByChannel)).Methods("GET")
-	api.BaseRoutes.Teams.Handle("/{team_id:[A-Za-z0-9]+}/groups", api.APILocal(getGroupsByTeam)).Methods("GET")
+	api.BaseRoutes.Channels.Handle("/{channel_id:[A-Za-z0-9]+}/groups", api.APILocal(getGroupsByChannelLocal)).Methods("GET")
+	api.BaseRoutes.Teams.Handle("/{team_id:[A-Za-z0-9]+}/groups", api.APILocal(getGroupsByTeamLocal)).Methods("GET")
+}
+
+func getGroupsByChannelLocal(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Do this function and the other one in this file contain the same code than the non-local versions except for the `requireLicense` call? If so, should we refactor them to a common helper function? 0/5

Reply:
Understood, on it.